perfkitbenchmarker/linux_benchmarks/fio_benchmark.py

```diff
@@ -124,10 +125,16 @@
                      'given number of minutes. Time will be rounded up to the '
                      'next multiple of %s minutes.' % MINUTES_PER_JOB,
                      lower_bound=0)
+flag_util.DEFINE_units('fio_blocksize', None,
+                       'The block size for fio operations. Default is given by '
+                       'the scenario when using --generate_scenarios and the '
```

Comment:
The help message here is not clear to me. When using custom job file, this flag is not used.